File: onlinejudge/baekjoon/Bronze/1152 단어의 개수.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String line = scan.nextLine();
        String[] words = line.split(" ");
        
        if (words.length == 0) {
            System.out.println(0);
        } else if (words[0] == "") {
            System.out.println(words.length - 1);
        } else {
            System.out.println(words.length);
        }
    }
}

File: onlinejudge/baekjoon/Bronze/1152 단어의 개수.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	var line string
	fmt.Scanln(&line)
	words := strings.Split(line, " ")

	if len(words) == 0 {
		fmt.Println(0)
	} else if words[0] == "" {
		fmt.Println(len(words) - 1)
	} else {
		fmt.Println(len(words))
	}
}
